JoonasHeinonen/escape-from-zordoom-prison
# Bugfix/dialogic 2

All the little issues that came with dialogic are now fixed



## Changes:
 Have so when you pull down the branch it makes it so you can talk with npc without much fuss.
-have it so that you can't shoot,move,or select weapons when talking to an npc or boss 
## How to test:
 -pull down the branch
-in the test level go to every npc in the level and talk with them as either Angela or Rivet.
- the boss should be able to talk with you and once they are done talking with you they attack you when the player starts moving,shooting etc

## Sources cited:
dialogic documention
Godot discord server
me messing around in dialogic.
